src/screens/CreatePost/elements/PollItem.js

```diff
@@ -1,24 +1,32 @@
 import * as React from 'react';
-import {Platform, StyleSheet, Text, TextInput, TouchableOpacity, View} from 'react-native';
+import {
+  Platform,
+  StyleSheet,
+  Text,
+  TextInput,
+  TouchableOpacity,
+  View,
+} from 'react-native';
 
 import MemoIcClearCircle from '../../../assets/icons/ic_clear_circle';
 import {MAX_POLLING_CHARACTER_ALLOWED} from '../../../utils/constants';
-import {COLORS} from '../../../utils/theme';
+import {colors} from '../../../utils/colors';
 
 export default function PollItem({
   index = 0,
   poll,
   onremovepoll = () => {},
   onpollchanged = () => {},
   showdeleteicon,
-  showcharactercount = false
+  showcharactercount = false,
 }) {
   const [isTextInputFocus, setIsTextInputFocus] = React.useState(false);
 
   return (
-    <View style={isTextInputFocus ? S.focuspollitemcontainer : S.pollitemcontainer}>
+    <View
+      style={isTextInputFocus ? S.focuspollitemcontainer : S.pollitemcontainer}>
       <TextInput
-        placeholderTextColor={COLORS.blackgrey}
+        placeholderTextColor={'#828282'}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The placeholder text color has been removed in the new hunk. If this was intentional and you want to use the default placeholder text color, then it's fine. Otherwise, you might want to add it back.
```diff
- placeholderTextColor={COLORS.blackgrey}
+ placeholderTextColor={'#828282'}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->